src/DB/exerciseData.json

```diff
@@ -249,4 +249,16 @@
     "gh-name": "haldaranup",
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2021/10/t-bar-row-muscles.gif"
   }
+  {
+    "exercise": "Heel Touches",
+    "instructions": [
+     "Laying flat on your back, place your hands behind your head. Bend your knees and bring them up into a 90-degree angle.",
+     "then lift into a crunch. Try to touch your heels to the ground while maintaining a 90 degree position in your legs and holding the static crunch.",
+     "Then, lift your legs back to the tabletop position as you breathe out. Consider.",
+     "adding ankle weights to make this move more challenging once you establish the proper form."
+    ],
+    "videoLink": "https://www.youtube.com/watch?v=PtX0z5_6sdM",
+    "gh-name": "https://github.com/Gallantfarhan",
```

Comment:
The convention for gh-name is just the GitHub username and is not the entire url. Could you please update that. Thank you.

Reply:
oh I thought it was a GitHub entire URL